cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/rest/CloudControllerRestClientImpl.java

```diff
@@ -2972,16 +2973,16 @@ public CloudControllerRestClientHttpRequestFactory(ClientHttpRequestFactory dele
         public ClientHttpRequest createRequest(URI uri, HttpMethod httpMethod) throws IOException {
             ClientHttpRequest request = delegate.createRequest(uri, httpMethod);
 
-            String authorizationHeader = oauthClient.getAuthorizationHeader();
-            if (authorizationHeader != null) {
-                request.getHeaders()
-                    .add(AUTHORIZATION_HEADER_KEY, authorizationHeader);
-            }
-
-            if (cloudCredentials != null && cloudCredentials.getProxyUser() != null) {
-                request.getHeaders()
-                    .add(PROXY_USER_HEADER_KEY, cloudCredentials.getProxyUser());
-            }
+//            String authorizationHeader = oauthClient.getAuthorizationHeader();
+//            if (authorizationHeader != null) {
+//                request.getHeaders()
+//                       .add(AUTHORIZATION_HEADER_KEY, authorizationHeader);
+//            }
+//
+//            if (cloudCredentials != null && cloudCredentials.getProxyUser() != null) {
+//                request.getHeaders()
+//                       .add(PROXY_USER_HEADER_KEY, cloudCredentials.getProxyUser());
+//            }
```

Comment:
same here

Reply:
All the code in the java client is "dead code", only the interface is used